From 09d95dd9f3dbae47735c083bff7ef1a0acea06b9 Mon Sep 17 00:00:00 2001
From: Milian Wolff <milian.wolff@kdab.com>
Date: Tue, 21 Jun 2022 11:44:06 +0200
Subject: [PATCH 164/165] Fix compile of tst_qimage in Qt5

QVector isn't the same as QList there yet, so we have to use QVector
in the test, otherwise it won't find any function to call here.

Change-Id: I9d0585907e6e38996119074247f63cee6cd03e20
---
 tests/auto/gui/image/qimage/tst_qimage.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/auto/gui/image/qimage/tst_qimage.cpp b/tests/auto/gui/image/qimage/tst_qimage.cpp
index 69d713f1ca..01971ee3c1 100644
--- a/tests/auto/gui/image/qimage/tst_qimage.cpp
+++ b/tests/auto/gui/image/qimage/tst_qimage.cpp
@@ -3572,7 +3572,7 @@ void tst_QImage::metadataPassthrough()
     QCOMPARE(converted.dotsPerMeterY(), a.dotsPerMeterY());
     QCOMPARE(converted.devicePixelRatio(), a.devicePixelRatio());
 
-    QList<QRgb> clut({ 0xFFFF0000, 0xFF00FF00, 0xFF0000FF });
+    QVector<QRgb> clut({ 0xFFFF0000, 0xFF00FF00, 0xFF0000FF });
     QImage convertedWithClut = a.convertToFormat(QImage::Format_Indexed8, clut);
     QCOMPARE(convertedWithClut.text(QStringLiteral("Test")), a.text(QStringLiteral("Test")));
     QCOMPARE(convertedWithClut.dotsPerMeterX(), a.dotsPerMeterX());
-- 
2.36.1

